changelogs/fragments/setup_filter_smarter.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - setup module, fix filter to adjust for missing ansible_ prefix on query.
```

This is tripping up the sanity test because it things the trailing underscore is an RST link.
```suggestion
  - setup module, fix filter to adjust for missing ``ansible_`` prefix on query.
```